R/chart.TimeSeries.R

```diff
@@ -146,169 +146,139 @@
 #'                  event.color = "red", lwd = 2)
 #' 
 #' @export
-chart.TimeSeries <-
```

Comment:
Running this fails for me: chart.TimeSeries(edhec, plot_engine = "plotly")

With the following error: 
'Error in layout(., title = main) : unused argument (title = main)'